src/api/helpers/get-populated-application/index.ts

```diff
@@ -84,6 +84,15 @@ const getPopulatedApplication = async (context: Context, application: KeystoneAp
     throw new Error(generateErrorMessage('business', application.id));
   }
 
+  const businessContactDetail = await context.db.BusinessContactDetail.findOne({
+    where: { id: business?.businessContactDetailId },
+  });
+
```

Comment:
Can we throw an error if `businessContactDetail` does not exist, like the others?